test/e2e/streaming_test.go

```diff
@@ -78,6 +77,45 @@ func (suite *StreamingTestSuite) TestStreaming() {
 	ProductionSmokeTest("simple-streaming")
 }
 
+func (suite *StreamingTestSuite) TestStreamingWithTLS() {
+	if !usingOLM {
+		t.Skip("This test should only run when using OLM")
+	}
+	// Make sure ES and the kafka instance are available
+	waitForElasticSearch()
+	waitForKafkaInstance()
+
+	kafkaUserName := "my-user"
+	kafkaUser := getKafkaUser(kafkaUserName, kafkaNamespace)
+	err := fw.Client.Create(context.TODO(), kafkaUser, &framework.CleanupOptions{TestContext: ctx, Timeout: timeout, RetryInterval: retryInterval})
```

Comment:
`TODO()` implies that it will change in the future to an appropriate value. As we don't have a context that we could use, the most appropriate here would be `context.Background()`.